cmd/promtool/main.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/prometheus/prometheus/config"
 	"github.com/prometheus/prometheus/pkg/rulefmt"
 	"github.com/prometheus/prometheus/promql"
+	configUtil "github.com/prometheus/prometheus/util/config"
```

Comment:
Though it looks a bit un-Go-ey, we've always snake_cased multi-word package names, so it'd be good to stay consistent with that. I.e. `configUtil` -> `config_util` and `yamlUtil` -> `yaml_util`, etc.